2024-07-25 13:07:49,094 ERROR frappe New Exception collected in error log
Site: row.localhost
Form Dict: {}
2024-07-25 13:22:27,953 ERROR frappe Could not take error snapshot: No module named 'payments'
Site: row.localhost
Form Dict: {}
Traceback (most recent call last):
  File "/home/karthik/Desktop/work/frappe-bench/apps/frappe/frappe/app.py", line 99, in application
    init_request(request)
  File "/home/karthik/Desktop/work/frappe-bench/apps/frappe/frappe/app.py", line 172, in init_request
    frappe.init(site=site, sites_path=_sites_path, force=True)
  File "/home/karthik/Desktop/work/frappe-bench/apps/frappe/frappe/__init__.py", line 312, in init
    setup_module_map(include_all_apps=not (frappe.request or frappe.job or frappe.flags.in_migrate))
  File "/home/karthik/Desktop/work/frappe-bench/apps/frappe/frappe/__init__.py", line 1738, in setup_module_map
    for module in get_module_list(app):
  File "/home/karthik/Desktop/work/frappe-bench/apps/frappe/frappe/__init__.py", line 1586, in get_module_list
    return get_file_items(get_app_path(app_name, "modules.txt"))
  File "/home/karthik/Desktop/work/frappe-bench/apps/frappe/frappe/__init__.py", line 1551, in get_app_path
    return get_pymodule_path(app_name, *joins)
  File "/home/karthik/Desktop/work/frappe-bench/apps/frappe/frappe/__init__.py", line 1581, in get_pymodule_path
    return abspath(join(dirname(get_module(scrub(modulename)).__file__ or ""), *joins))
  File "/home/karthik/Desktop/work/frappe-bench/apps/frappe/frappe/__init__.py", line 1522, in get_module
    return importlib.import_module(modulename)
  File "/usr/lib/python3.10/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1004, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'payments'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/karthik/Desktop/work/frappe-bench/apps/frappe/frappe/utils/error.py", line 87, in log_error_snapshot
    log_error(title=str(exception), defer_insert=True)
  File "/home/karthik/Desktop/work/frappe-bench/apps/frappe/frappe/utils/error.py", line 62, in log_error
    error_log = frappe.get_doc(
  File "/home/karthik/Desktop/work/frappe-bench/apps/frappe/frappe/__init__.py", line 1379, in get_doc
    doc = frappe.model.document.get_doc(*args, **kwargs)
  File "/home/karthik/Desktop/work/frappe-bench/apps/frappe/frappe/model/document.py", line 85, in get_doc
    controller = get_controller(doctype)
  File "/home/karthik/Desktop/work/frappe-bench/apps/frappe/frappe/model/base_document.py", line 67, in get_controller
    return import_controller(doctype)
  File "/home/karthik/Desktop/work/frappe-bench/apps/frappe/frappe/model/base_document.py", line 89, in import_controller
    class_overrides = frappe.get_hooks("override_doctype_class")
  File "/home/karthik/Desktop/work/frappe-bench/apps/frappe/frappe/__init__.py", line 1687, in get_hooks
    hooks = _dict(_load_app_hooks())
  File "/home/karthik/Desktop/work/frappe-bench/apps/frappe/frappe/utils/caching.py", line 47, in wrapper
    return func(*args, **kwargs)
  File "/home/karthik/Desktop/work/frappe-bench/apps/frappe/frappe/__init__.py", line 1656, in _load_app_hooks
    app_hooks = get_module(f"{app}.hooks")
  File "/home/karthik/Desktop/work/frappe-bench/apps/frappe/frappe/__init__.py", line 1522, in get_module
    return importlib.import_module(modulename)
  File "/usr/lib/python3.10/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 992, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1004, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'payments'
